### PR TITLE
Update regnet.py

### PR DESCRIPTION
**Pull request by timothylimyl**
## Motivation

In the example comment to print out the different layers of outputs, we need to indicate the `out_indices` to (0,1,2,3) to see all backbone layers output as the default argument is (3,).

Let's just ensure that the user won't be confused on why the output printed does not match the code in the comments.

## Modification

Just adding out_indices argument call to output all layers instead of leaving it to default.

### CODE REVIEW

**Review by Ezra-Yu — COMMENTED** [review]
Thank you for your report.

It would be better if the case of str could be added, Can you please help us update it in this PR.

```
>>> from mmcls.models import RegNet
>>> import torch
>>> inputs = torch.rand(1, 3, 32, 32)
>>> # use str type 'arch'
>>> regnet_cfg = dict(arch='regnetx_4.0gf')  # default out_indices is (3,)
>>> model = RegNet(**regnet_cfg)
>>> model.eval()
>>> level_outputs = model(inputs)
>>> for level_out in level_outputs:
...     print(tuple(level_out.shape))
... 
(1, 1360, 1, 1)
>>> 
>>> # use dict type 'arch'
>>> arch_cfg =dict(w0=88, wa=26.31, wm=2.25, group_w=48, depth=25, bot_mul=1.0)
>>> regnet_cfg = dict(arch=arch_cfg, out_indices=(0, 1, 2, 3))
>>> model = RegNet(**regnet_cfg)
>>> model.eval()
>>> level_outputs = model(inputs)
>>> for level_out in level_outputs:
...     print(tuple(level_out.shape))
... 
(1, 96, 8, 8)
(1, 192, 4, 4)
(1, 432, 2, 2)
(1, 1008, 1, 1)
````

**Comment by timothylimyl** [comment]
@Ezra-Yu follow your changes exactly.

**Comment by mzr1996** [comment]
> @Ezra-Yu follow your changes exactly.

Please fix the lint error before merge.

**Comment by timothylimyl** [comment]
@mzr1996 done, sorry for the multiple useless commits, was not aware of the patch branch. You can see that the file changes is still just `regnet.py` docs.

**Comment by timothylimyl** [comment]
ping @Ezra-Yu , can close?

File: mmcls/models/backbones/regnet.py [hunk 1]
```diff
@@ -45,23 +45,32 @@ class RegNet(ResNet):
...
+        >>> model.eval()
+        >>> level_outputs = model(inputs)
         >>> for level_out in level_outputs:
         ...     print(tuple(level_out.shape))
-        (1, 96, 8, 8)
-        (1, 192, 4, 4)
-        (1, 432, 2, 2)
-        (1, 1008, 1, 1)
+        ...
+            (1, 1360, 1, 1)
```
Comment (Ezra-Yu) [inline mmcls/models/backbones/regnet.py:58]: I am sorry for  later reply.
There are still some errors.
image

please remove the ``·blank line```

you can check the update in the link
image

**Comment by Ezra-Yu** [comment]
> ping @Ezra-Yu , can close?

We can finish the PR today.

**Review by Ezra-Yu — APPROVED** [review]
LGTM.

**Comment by timothylimyl** [comment]
docs is till off, I can't figure out how does the formatting works for now

**Comment by Ezra-Yu** [comment]
> docs is still off, I can't figure out how does the formatting works for now

can you open this [link](https://mmclassification--935.org.readthedocs.build/zh_CN/935/api/generated/mmcls.models.RegNet.html#mmcls.models.RegNet)

image

**Comment by timothylimyl** [comment]
thanks.

**Pull request merged**